lib/src/client.dart

```diff
@@ -45,92 +50,43 @@ class TusClient {
   Future? _chunkPatchFuture;
 
   TusClient(
-    this.url,
-    this.file, {
+    this.url, {
+    this.file,
+    this.fileStream,
```

Comment:
Not quite sure why this is needed. Can the stream not be wrapped before passing it?

```dart
XFile file = XFile.fromData(fileStream);
```